packages/libraries/cli/src/helpers/errors.ts

```diff
@@ -0,0 +1 @@
+export const ACCESS_TOKEN_MISSING = `--registry.accessToken is required in order to perform this command on the schema registry. To create an registry access token with the necessary permissions, use the Hive console. Select the target under your Hive project and then choose the Settings tab. On the Settings screen, you can manage your target's settings and access tokens. Under the Registry Access Tokens section, click the Create new registry token button. Give your access token a name and select Read & Write for the permissions.`;
```

Comment:
I think we can add a link to the docs? Maybe to this page: https://the-guild.dev/graphql/hive/docs/management/targets#registry-access-tokens ?

Reply:
I mean, instead of having a long CLI message, we can have a short, concise message, with a link to the docs.

Reply:
I've adjusted the message to reflect this.